package.json

```diff
@@ -31,9 +31,12 @@
     "**/*": "prettier --write --ignore-unknown"
   },
   "dependencies": {
+    "@headlessui/react": "^1.7.7",
     "@next-auth/prisma-adapter": "^1.0.5",
     "@next/font": "^13.1.1",
     "@prisma/client": "^4.8.0",
+    "@radix-ui/react-dialog": "^1.0.2",
+    "@radix-ui/react-icons": "^1.1.1",
```

Comment:
Uninstall this package. copy the SVG from https://icons.radix-ui.com.Then, paste it in icons folder and export-import it from there

Reply:
done